src/app/hearing/components/view-hearing/view-hearing.component.html

```diff
@@ -0,0 +1,120 @@
+<mat-card *ngIf="hearing">
+  <h2>{{hearing.caseTitle}}</h2>
+  <h3>{{hearing.caseNumber}}</h3>
+  <div class="grid">
+    <div class="tile">
+      <strong>Case type</strong><br>
+      {{hearing.caseType}}
+    </div>
+
+    <div class="tile">
+      <strong>Hearing Type</strong><br>
+      {{hearing.hearingType}}
+    </div>
+
+    <div class="tile">
+      <strong>Priority</strong><br>
+      {{hearing.priority}}
+    </div>
+
+    <div class="tile">
+      <strong>Estimated duration</strong><br>
+      {{formatDuration(hearing.duration)}}
+    </div>
+
+    <div class="tile">
+      <strong>List between</strong><br>
+      {{formatDate(hearing.scheduleStart)}} - {{formatDate(hearing.scheduleEnd)}}
+    </div>
+
+    <div class="tile">
+    </div>
+
+    <div class="tile">
+      <strong>Reserved for</strong><br>
+      {{hearing.reservedToJudge}}
+    </div>
+
+    <div class="tile">
+      <strong>Communication facilitator</strong><br>
+      {{hearing.communicationFacilitator}}
+    </div>
+
+    <div class="tile">
+    </div>
+
+    <div class="tile">
+      <strong>Special requirements</strong><br>
+      {{hearing.specialRequirements}}
+    </div>
+
+    <div class="tile">
+      <strong>Facility requirements</strong><br>
+      {{hearing.facilityRequirements}}
+    </div>
+
+    <div class="clear"></div>
+  </div>
+  <mat-tab-group>
+
+    <mat-tab label="Notes">
+      <mat-list class="hearing-notes">
+        <mat-list-item *ngFor="let note of hearing.notes" class="note">
+          <div class="content">
+            {{note.content}}
+          </div>
+          <div class="author-info">
+            <strong>{{note.modifiedBy}}</strong><br>
+            {{formatDate(note.createdAt)}}
+          </div>
+        </mat-list-item>
+      </mat-list>
+    </mat-tab>
+
+    <mat-tab label="Scheduled listings">
+      <mat-accordion>
+        <mat-expansion-panel [expanded]="true" *ngFor="let session of hearing.sessions">
+          <mat-expansion-panel-header>
+            <mat-panel-title>
+              <div class="listing-header">
+                <div class="tile">
+                  {{formatDate(session.start)}}
+                </div>
+                <div class="tile">
+                  {{session.sessionType}}
+                </div>
+                <div class="tile">
+                  Start {{formatDate(session.start)}}
+                </div>
+                <div class="tile">
+                  {{session.duration.toString()}} minutes
```

Comment:
Are there any differences between **session.duration** and **hearing.duration**?
What I can see we are handling them in a quite different way.